test/api/v3/integration/groups/GET-groups.test.js

```diff
@@ -0,0 +1,51 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('GET /groups', () => {
+  let user;
+
+  beforeEach(async () => {
+    user = await generateUser({balance: 4});
+  });
+
+  it('returns a list of groups', async () => {
+    let groups = [
+      {
+        name: 'Test Party',
+        type: 'party',
+      },
+      {
+        name: 'Test Public Guild',
+        type: 'guild',
+      },
+      {
+        name: 'Test Private Guild',
+        type: 'guild',
+        privacy: 'private',
+      },
+    ];
+
+    let group1 = await user.post('/groups', {
```

Comment:
@TheHollidayInn forEach did produce an error because when you use await the direct parent function must be `async function () {}` not a normal `function () {}`